test/cypress/tests/onchange.cy.ts

```diff
@@ -1,5 +1,6 @@
 import Header from '@editorjs/header';
 import Code from '@editorjs/code';
+import Checklist from '@editorjs/checklist';
```

Comment:
Is this dependency really required? It would be better to use mock instead, check `fixtures/tools/ToolMock.ts`

Reply:
Replaced with a ToolMock